ministryofjustice/analytics-platform-control-panel-frontend
# Fetch Github repos from Github API

## What

* Added Github API client and model classes
* Fetch Github repos from create app handler (with user's Github access token) and pass to template
* Update create app form to show a dropdown of available Github repos (including private)

## How to review

1. Go to http://localhost:3000/apps/new
2. See the "Github repository URL" form field group has a second dropdown of repo names

See [Trello #509](https://trello.com/c/k4rsqehc)
